src/main/java/com/mlaf/hu/implementationexample/sensoragent/SensorAgent.java

```diff
@@ -0,0 +1,123 @@
+package com.mlaf.hu.implementationexample.sensoragent;
+import com.mlaf.hu.sensoragent.InvalidSensorException;
+
+import java.util.logging.Level;
+
+public class SensorAgent extends com.mlaf.hu.sensoragent.SensorAgent {
```

Comment:
Ik ben niet zo'n fan van duplicate names, ik zou deze dan ExampleServiceAgent noemen zodat je niet zo'n debiele extends hoeft te doen

Reply:
@rogierpijpers  Wat is jouw mening hierover? Een collega op werk heeft Auke namelijk al ongelijk gegeven. Het package system is hier eigenlijk voor gemaakt. Ander zouden we alles in 1 grote map kunnen gooien en het allemaal een andere naam geven.

Reply:
Is toch super verwarrend. Oke welke sensor agent is dit? die van de package of die van die andere?
Ik neem aan dat je niet de hele tijd com.mlaf.hu.sensoragent. ervoor gaat typen

Reply:
Ik ben t wel met Auke eens. Duplicate names mogen wat mij betreft wel bestaan, maar dan meer als ik het over een Parser in verschillende packages heb ofzo. In dit geval vind ik het wat verwarrend. Maar echt uitmaken doet t verder niet:P